### PR TITLE
Add create simple on-chain blog tutorial

### PR DESCRIPTION
**Pull request by ahonn**

### CODE REVIEW

**Review by ShookLyngs — APPROVED** [review]

**Review by code-monad — APPROVED** [review]

File: docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 1
+---
+
+# Creating simple on-chain blog
```
Comment (anipenguin) [inline docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx:5]:
```suggestion
# Creating Simple On-Chain Blog
```

File: docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Getting started
```
Comment (anipenguin) [inline docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx:7]:
```suggestion

```

File: docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+This tutorial will guide you through the process of using the Spore Protocol to create an on-chain blog. With the Spore Protocol, you can store your blog posts fully on the blockchain and allow others to access them. Following my footsteps will realize a simple on-chain blog, just like the online demo I deployed here: [https://spore-blog-tutorial.vercel.app](https://spore-blog-tutorial.vercel.app/)
```
Comment (anipenguin) [inline docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx:9]:
```suggestion
This tutorial will guide you through the process of using the Spore Protocol to build an on-chain blog. By the end, you'll have your own basic on-chain blog, similar to the one demonstrated here: [https://spore-blog-tutorial.vercel.app](https://spore-blog-tutorial.vercel.app/)
```

File: docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Here is the code repository that goes along with this tutorial: [https://github.com/sporeprotocol/spore-blog-tutorial](https://github.com/sporeprotocol/spore-blog-tutorial). You can view or checkout different branches to see the code at different stages. I will also let you know which branch corresponds to each stage when it ends.
```
Comment (anipenguin) [inline docs/tutorials/2-create-simple-on-chain-blog-step-by-step/index.mdx:11]:
```suggestion
You can find the code repository for this tutorial at: [Spore Blog Tutorial Repository](https://github.com/sporeprotocol/spore-blog-tutorial). In this repository, you can explore different branches to see the code at various stages of development. We will clearly indicate which branch corresponds to each stage as we progress.
```

**Review by anipenguin — APPROVED** [review]
will upload infographic after merge

**Pull request merged**